### DISCUSSION TITLE
I think you guys should create a star rating component

### DISCUSSION THREAD

**Discussion by AnointingMax** [shadcn-ui/ui#5490] [Ideas]
Hello @shadcn,

I love your components, they have been a huge help in my projects. But don't you also need a star rating component?

Happy to help if there is also interest from other people.

Signed,
Just your very average react dev

**Comment by dennis2go** [comment]
fr fr

**Comment by Steveb599** [comment]
https://github.com/shadcn-ui/ui/issues/1107

**Reply by AnointingMax** [reply]
Thank you. 
I searched for an issue concerning it and didn't find anything. I guess I might not have searched properly
